scripting/get5/recording.sp

```diff
@@ -136,3 +198,32 @@ void SetCurrentMatchRestartDelay(float delay) {
     mp_match_restart_delay.FloatValue = delay;
   }
 }
+
+public int DemoRequestCallback(Handle request, bool failure, bool requestSuccessful,
+                        EHTTPStatusCode statusCode) {
+  int responseSize;
+  SteamWorks_GetHTTPResponseBodySize(request, responseSize);
+  char[] response = new char[responseSize];
+  SteamWorks_GetHTTPResponseBodyData(request, response, responseSize);
```

Comment:
When before the `failure` check, you must test if this works for network connection errors, which produce no HTTP response body.

Reply:
Per SteamWorks docs, it appears we can just check that as well with an `if` - 
> Returns: bool
Returns true upon success indicating that pBodyDataBuffer has been filled with the body data.
Otherwise, returns false under the following conditions:
hRequest was invalid.
The request has not been sent or has not completed.
The request is a streaming request.
pBodyDataBuffer is NULL.

Where pBodyDataBuffer is our `response` variable, if I'm reading this correctly?

Reply:
Yes, that seems possible.